### DISCUSSION TITLE
where can I find XSDGen.exe

### DISCUSSION THREAD

**Discussion by john1726** [pocoproject/poco#3387] [Q&A]
Please note that I am following along with this guide:

https://pocoproject.org/pro/docs/00152-RemotingNGTutorialPart3.html

Here it discusses using XSDGen.  Where can I find this tool on Windows 10?  Here are some sites I have looked at:

https://www.programmerall.com/article/1350818817/
https://github.com/wiztools/xsd-gen

Also, I have found this:

https://github.com/AnalogJ/Wix3.6Toolset/blob/master/RC0-source/wix36-sources/toolsrc/WixBuild/XsdGen.cs

But when I run this tool it takes different flags than what are explained in the tutorial:

```
C:\Projects>XsdGen.exe /config=XSDGen.xml http://ec.europa.eu/taxation_customs/vies/checkVatService.wsdl
usage: XsdGen.exe <schema>.xsd <outputFile> <namespace> [<commonNamespace>]	
```

Also, I don't think these tools are the same as the one discussed in the Remoting NG Tutorial Part 3.  Do you have any suggestions?  TIA.

**Comment by aleks-f** [comment]
that's part of [POCOpro](https://pocoproject.org/pocopro.html)
